swp/migrations/0040_publication_publication_dt.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 3.2.16 on 2023-04-13 14:56
+
+from django.db import migrations, models
+
+from swp.utils.date import parse_date
+
+
+def migrate_date(apps, schema_editor):
+    Publication = apps.get_model('swp', 'Publication')
+
+    for publication in Publication.objects.all():
+        default = publication.created.date().isoformat()
+        publication.publication_dt = parse_date(publication.publication_date, default_date_str=default)
+        publication.save(update_fields=['publication_dt'])
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('swp', '0039_publication_list'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='publication',
+            name='publication_dt',
+            field=models.DateField(blank=True, null=True, verbose_name='publication dt'),
+        ),
+        migrations.RunPython(code=migrate_date),
```

Comment:
Consider adding a reverse migration. In this case a `migrations.RunPython.noop` is sufficient. But with out it, this migration cannot be reverted.